### ISSUE TITLE
Make react-intl optional

### ISSUE THREAD

**Issue by s7dhansh** [ssr-server/ssr#61]
Is there a proper way to do it? From the looks of it, it seems embedded deep in the code. I can easily remove `i18n` but not `react-intl`/`react-intl-redux` 

Basically with Meteor 1.5, `bundle-visualizer` shows intl packages to be taking almost 59% (1.4mb) of a small app.

I also saw https://ssrwpo.github.io/ssr/i8n/ but to no fruitful avail.

**Comment by PEM--** [comment]
Nice found. A PR would be great. I'm changing the title to a less confusing meaning as this is the `develop` branch.

**Comment by s7dhansh** [comment]
Will try. But I am not familiar with the code-base at all. Some basic pointers would be helpful. As in why do you use it (what kind of messages and at what steps). 

From the code, I feel just changing the `Provider` to be taken from `react-redux` instead pf `react-intl-redux` should do the trick. The other `intl` related files are the reducers and actions, which appear to be harmless. Though, we should make them optional somehow.

**Comment by cbilotta** [comment]
Can you give me an email address so I can invite you to slack

**Comment by s7dhansh** [comment]
sudhansh AT gmail

**Comment by cbilotta** [comment]
Have you received my invitation ?
Use it for basically anything - But try to talk in private if you get caught up in needing explanations instead of spamming the general channel :)

**Comment by s7dhansh** [comment]
Yes I have. Thank you. I will try to get my doubts cleared on Monday. Weekends are quite hectic for me :)

**Comment by sakulstra** [comment]
@s7dhansh is there any progress on this? can i somehow help? - i tried your simple approach (replacing the provider) and for me it seems to work (but i'm not sure if it won't break for intl users)

**Comment by s7dhansh** [comment]
@sakulstra Sorry no. I did not get enough time to do this. For now I use my own fork where I just remove everything. Not ideal, just works for me now as I do not need `intl`.
You may get cues here https://github.com/settlin/ssr/tree/settlin.

**Comment by cbilotta** [comment]
Updates on this ?
